### ISSUE TITLE
"I wonder if you can see me Mr.Kennedy" cutscene really likes to crash. 

### ISSUE THREAD

**Issue by Deus-nsf** [nipkownix/re4_tweaks#566]
### Validation

- [X] Game has been updated to the latest RE4HD release (v1.1: https://www.re4hd.com/?p=9552)
- [X] Data has been [validated with SFV](https://www.re4hd.com/?p=9454) (a guide to using QuickSFV is available at that link, use that with this 1.1 SFV: [BIO4-HDProject1.1-SFV.zip](https://github.com/emoose/re4-research/files/9207370/BIO4-HDProject1.1-SFV.zip))

### Describe your issue here (drag+drop ZIP to attach it)

The game really likes to crash here, tested on different computers with completely different hardware (Nvidia or AMD GPU) and with shorter game sessions too: https://www.youtube.com/live/Y0sabYiLeq0?feature=share&t=18768

**Comment by Deus-nsf** [comment]
I tried to disable the models pre-cache thing and the multithread fix, since I had those enabled, maybe that will help.

**Comment by nipkownix** [comment]
Please upload a CrashDump if you can. Instructions:

>try creating a CrashDumps folder inside the Bin32 directory (next to bio4.exe/steam_api.dll/etc), then the next time game crashes hopefully some info about it will be written in there

**Comment by linkthehylian** [comment]
Yep, this is the exact cutscene that I had crashing issues with. I made an issue for it (https://github.com/nipkownix/re4_tweaks/issues/431) back in January when doing a Professional run.

**Comment by Deus-nsf** [comment]
> Yep, this is the exact cutscene that I had crashing issues with. I made an issue for it (#431) back in January when doing a Professional run.

This has to be memory related. If only all things were 64 bit...
It only happens with continuous runs, playing the game in short sessions seems to be fine.
For the time being I always save the game just before that part and reboot/load the game, seems to be fine that way.

**Comment by linkthehylian** [comment]
> This has to be memory related.
> It only happens with continuous runs, playing the game in short sessions seems to be fine.

It definitely has to be. Shorter sessions (such as immediately jumping to the room) like you said never cause it to happen at all.

I think this is literally the only issue plaguing the most recent releases of this project. **Please** provide your crash dumps if it happens so that this can be fixed faster. Follow @nipkownix's instructions (https://github.com/nipkownix/re4_tweaks/issues/566#issuecomment-1574903106) on how to provide a dump if you don't know how.

**Comment by linkthehylian** [comment]
Managed to get it to crash again right outside the clock tower. Here you go @nipkownix.

[bio4.exe.20230621205012.zip](https://github.com/nipkownix/re4_tweaks/files/11827517/bio4.exe.20230621205012.zip)

Something I discovered regarding this too, but you actually don't have to play until you reach the clock tower. If you progress until you reach the village and then area jump to it, it'll still crash. Very interesting.

**Edit**: Unsure if it's related (it probably is lol), but I also crashed after trying to insert the medallions after fighting Krauser. (https://github.com/nipkownix/re4_tweaks/issues/431#issuecomment-1379766723) and after rescuing Ashley near the end of the game (https://github.com/nipkownix/re4_tweaks/issues/431#issuecomment-1470958922)

emoose had me try a specific build as well but I think he got busy and wasn't able to investigate it after.
